docs/pages/datapoint.js

```diff
@@ -39,9 +39,70 @@ export default function DatapointPage({ generatedDocGen }: {| generatedDocGen: D
           />
         </MainSection.Subsection>
       </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Whenever possible, use the full number with separators to ensure clarity across all languages."
```

Comment:
What would be the opposite of this one? 

_Whenever possible, use the full number with separators to ensure clarity across all languages._

I'm not sure I understand this one.

Reply:
Added some more detail to this guideline to hopefully clarify.